server/routes/v1/deployment.ts

```diff
@@ -257,6 +260,131 @@ export const fetchRawModelFiles = [
   },
 ]
 
+type DirectoryMetadata = {
+  name: string
+  directories: Map<string, DirectoryMetadata>
+  files: string[]
+}
+
+const addToTree = (tree: DirectoryMetadata | undefined, path: string): void => {
+  if (tree === undefined) {
+    throw new Error('Error: Tree does not exist')
+  }
+  if (path.includes('/')) {
+    const i = path.indexOf('/')
```

Comment:
Rename to index

Reply:
That's a javascript function, not a variable name

Reply:
I think he meant changing the `i` to `index`